soot-infoflow/pom.xml

```diff
@@ -8,11 +8,45 @@
 	<version>2.7.2-SNAPSHOT</version>
 	<description>Soot extending data flow tracking components for Java</description>
 	<packaging>jar</packaging>
+	<url>https://github.com/secure-software-engineering/FlowDroid</url>
+    <organization>
+        <name>Fraunhofer SIT</name>
+        <url>https://www.sit.fraunhofer.de/</url>
+    </organization>
+    <licenses>
+        <license>
+            <name>GNU LESSER GENERAL PUBLIC LICENSE 2.1</name>
+            <url>https://www.gnu.org/licenses/lgpl-2.1.txt</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+    <developers>
+        <developer>
+            <name>Steven Arzt</name>
+            <email>steven.arzt@sit.fraunhofer.de</email>
+            <organization>Frauenhofer SIT</organization>
```

Comment:
Typo, without the "e"

Reply:
ohh good to know, we have the same typo in Soot.